src/main/resources/com/microsoft/jenkins/containeragents/aci/AciCloud/help-resourceGroup.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    Specify the resource group. ACI is only supported in West US, East US or West Europe.
+    Specify the resource group.
```

Comment:
Also add the [Products available by region](https://azure.microsoft.com/en-us/global-infrastructure/services/?products=container-instances)

Reply:
done